project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/layout/header/search.less

```diff
@@ -7,35 +7,21 @@
     &__form {
         display: flex;
 
-        .is-no-flex & {
-            display: block;
-            .clearfix();
-        }
-
         &__input {
-            flex: 1;
+            width: ~"calc(100% - @{search-button-width})";
```

Comment:
Please check if you can substitute `flex` property for the `width` property.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width

Reply:
It is okay for now. We will refactor this later.